packages/web-twig/src/Resources/components/Dropdown/README.md

```diff
@@ -47,17 +47,25 @@ attributes to register trigger events.
 
 ### Dropdown
 
-| Name            | Type                                         | Default       | Required | Description                           |
-| --------------- | -------------------------------------------- | ------------- | -------- | ------------------------------------- |
-| `elementType`   | `string`                                     | `div`         | ✕        | HTML tag to render                    |
-| `fullWidthMode` | `string`                                     | —             | ✕        | Full-width mode [off,mobile-only,all] |
-| `id`            | `string`                                     | —             | ✔        | Dropdown ID                           |
-| `placement`     | [Placement Dictionary][dictionary-placement] | `bottom-left` | ✕        | Placement of the dropdown             |
+| Name            | Type                                         | Default        | Required | Description                           |
+| --------------- | -------------------------------------------- | -------------- | -------- | ------------------------------------- |
+| `elementType`   | `string`                                     | `div`          | ✕        | HTML tag to render                    |
+| `fullWidthMode` | `string`                                     | —              | ✕        | Full-width mode [off,mobile-only,all] |
+| `id`            | `string`                                     | —              | ✔        | Dropdown ID                           |
+| `placement`     | [Placement Dictionary][dictionary-placement] | `bottom-start` | ✕        | Placement of the dropdown             |
 
 On top of the API options, the components accept [additional attributes][readme-additional-attributes].
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
+#### ⚠️ DEPRECATION NOTICE
+
+Both axis side placements are renamed from `top-left`, `top-right`, `right-top`, `right-bottom`,
```

Comment:
I am not fluent in English, but `both axis side placements` sounds weird to me and I must think hard about what it means. Maybe `side placements on bot axis`